<html>
<head>
<title>Research Funding Awarded to Longhai Li</title>
</head>
<body>

<script src="load.js"></script>


<script>
$(function(){
  $("#nav").load("nav.html");
});
</script>
	<!--Navigation bar  -->
<div id="nav"></div>
<table width="100%" class="topnav">
<td width="40%" align="left" valign="middle">
 <img src="images/usask-logo-lg.png"; name="UofS Logo"; alt="UofS Logo"; align="left"; height="40"; border="0"/>&nbsp Prof. Longhai Li
</td>
<!-- <td width="22%" aligh="left">&nbsp Prof. Longhai Li</td> -->
<td width="60%" align="right">                   
        <a href="index.html"><button class="btn">About me</button></a> 
        <a href="teaching.html" ><button class="btn">Teaching</button></a> 
        <a href="grant.html" ><button class="active">Projects</button></a> 
        <a href="team.html" ><button class="btn">Students</button></a> 
        <a href="publications.html" ><button class="btn">Publications</button></a> 
        <a href="software.html" ><button class="btn">Software </button></a> 

</td>

</table>


<div class="main">	

<h2 align="left">Externally Funded Research Projects</h2>

<ul>
	<li>
<p style="margin-bottom: 0cm"> <b>Predictive Methods for Analyzing
	High-throughput and Spatial-temporal Data</b>, NSERC Individual Discovery Grant, 2019 -
	2024, PI. 
	</p>
	<ul>
		<li> <h4>Predictive Analysis for High-throughput Data </h4>
		<p/>The accelerated development of high-throughput sequencing biotechnologies has made it affordable to collect high-dimensional molecular-level profiles, such as gene expression, which are called features in general.  It is of great interest to identify relevant features associated with a phenotype (eg. cancer status, health disorder). Many researchers have advocated to apply statistical learning methods to perform predictive analysis for high-throughput data. Predictive analysis results can be used in many ways. For example, they can be used to diagnose human diseases, to predict response to a medicine (personalized medicine); they can be used to choose an optimal gene subset for further experiments by plant/animal breeders; the subset of features extracted from good predictive models can facilitate the uncovering of the biological mechanism for a phenotype. Unfortunately, the high-dimensionality causes enormous overfitting in predictive analysis even with very simple models. The chance of finding false predictive features/patterns is extremely high.  Therefore, it is challenging to fight against false discovery in predictive analysis.  My research in this theme aims to develop new tools for honestly measuring predictivity (such as error rate, AUC) of selected features, and new tools for identifying truly predictive features and for building sharper predictive models for phenotypes. I also practice predictive analysis with specific high-throughput datasets in a variety of scientific problems related to human health. 
		</li>
		<li> <h4>Predictive Model Evaluation Methods for Spatial-Temporal Data</h4>
		<p>In science, a theory is tested by performing 
predictions for observations in the future. Significant discrepancies 
between observations and predictions suggest that the theory is incorrect 
or flawed. Similarly, looking at out-of-sample predictions is a 
straightforward method for comparing and checking goodness-of-fit (GOF) of 
statistical models. Today, increasingly complex models are being proposed 
for a variety of correlated data such as, temporal, spatial, and repeated 
measurements data. More widely applicable predictive methods for comparing 
and checking such complex models are demanded. My research in this theme 
aims to develop new tools for evaluating complex Bayesian/non-Bayesian 
models with correlated random effects, with applications in many areas 
such as epidemiology, ecology, and environmental sciences. </li>
</p>
</li>
</ul>


	<li>
		<b>MITACS Accelerate Grant: Develop a web based geospatial artificial intelligence framework to track, visualize, analyze, model, and predict infectious disease spread in real-time</b>, PI, 2020-2021
	</li>


<li/>
<p style="margin-bottom: 0cm">
	<b>Genotype &amp; Environment to Phenotype</b>, sub-project from Canada First Research Excellence Fund (CFREF) Project &quot;Designing Crops for Global Food Security&quot;,  $756,918, 2016-2019, Co-Investigator (PI: Prof. Kusalik). 
</p>





<li/>
<p style="margin-bottom: 0cm">
	<b>Applications of Neural Network Curve
	Fitting Methods for Least-squares Monte Carlo Simulations in
	Financial Risk Management</b>, MITACS Accelerate Internship Fund, 2016, PI. 
</p>
<!--
<p>
	Monte Carlo simulation methods are commonly used as a risk management tool to estimate the risk exposure of financial asset portfolios. However, the traditional brute-force Monte Carlo (BFMC) method is often very timeconsuming, which makes it difficult to serve the risk management needs of modern insurance industry. An alternative approach, the least-squares Monte Carlo (LSMC) method, could substantially reduce the computation cost by fitting a proxy function of liabilities using simple nonlinear regression methods. However, the LSMC does not work well in capturing the true risk properties of hedged assets and some other variable annuities. To improve the LSMC method, we propose to use neural network method during the curve fitting process. Since the neural network method allows more curve-fitting flexibility, the proxy functions of liabilities and Greeks are expected to be improved. By using the improved proxy functions to predict the liabilities and Greeks, the risk exposure of financial portfolios can be more properly quantified.	
</p>
-->
	
<li/>
<p style="margin-bottom: 0cm">
	<b>Bayesian Methods for High-dimensional and Correlated Data</b>, NSERC Individual Discovery Grant,  2014 - 2019, PI.
</p>
	
<li/>
<p style="margin-bottom: 0cm">
	<b>Efficient Bayesian Analysis for Complex Models</b>, NSERC Individual Discovery Grant,  2009 - 2014, PI.
</p>
	
<li/>
<p style="margin-bottom: 0cm">
	<b>A Computer Cluster for Research on Efficient
	Bayesian Statistical Methods</b>, CFI Leaders Opportunity Fund,  2009, PI. 
</p>
	
<li/>
<p>
	<b>Clustering Analysis for Detecting the Types of Vehicles</b>, MITACS Accelerate Internship Fund,  2008, Co-PI with Prof. Laverty. 
</p>
<!--
	<p>
		It is very useful to build an automatic computer system to recognize the types of vehicles passing a checkpoint given some easy-to-get data about the vehicles, such as the distances between axles, the weights on each axle. Such a system has many applications, for example, in monitoring traffic volumes and identifies the type of vehicle, which will be helpful in budgeting road maintenance costs. The main goal of this project is to develop a better methodology for cluster analysis with application to the vehicle detection problem. The simplest clustering technique is the K-means clustering. However, K-means clustering requires that the users supply with a number of clusters. X-means clustering may be an alternative method since it can detect the number of clusters with some simple criteria. But X-means would introduce more severe local mode problem. We will investigate a new method to overcome the problem by merging similar clusters after running X-means clustering.
	</p>
-->	
</ul>



</div>
</body>
</html>
