<html>
<head>

<meta charset="utf-8">

<title>Research Funding Awarded to Longhai Li</title>
<link rel="icon" type="image/x-icon" href="/images/favicon.ico">
<link rel="stylesheet" href="mystyles.css">

</head>
<body>


<table width="100%" class="topnav">
	
<td style="min-width: 350px; width:30%;  align: left; valign: middle">
	<img src="images/usask-logo-lg.png"; style="height:40; margin-top:0px; margin-right:20px;"; name="UofS Logo"; align="left"; border=0; alt="UofS Logo";>Prof. Longhai Li
</td>

<td width="60%" align="right">                   
        <a href="index.html"><button class="btn">About me</button></a> 
        <a href="teaching.html" ><button class="btn">Teaching</button></a> 
        <a href="grant.html" ><button class="active">Projects</button></a> 
        <a href="team.html" ><button class="btn">Students</button></a> 
        <a href="publications.html" ><button class="btn">Publications</button></a> 
        <a href="software.html" ><button class="btn">Software </button></a> 

</td>

</table>


<div class="main">

<h2>Overview of Longhai Li's Research Activities </h2>


Prof. Li's research focuses on developing and applying statistical machine-learning methods to analyze bioinformatics and epidemiology data. His primary research areas include:

<ul>


<li> <h4>Model Diagnostics and Comparison</h4></li>

He aims to develop new tools for evaluating Bayesian/non-Bayesian models with complex structures. Today, increasingly complicated models are being proposed for a variety of correlated data such as temporal, spatial, and repeated measurement data. There is a gap between developing new modelling methods and model validation methods.  He is working on developing new residual diagnostic methods for checking the adequacy of statistical models. 


<li><h4>Statistical Machine Learning</h4></li>


He aims to develop new tools for honestly measuring the predictivity (such as error rate, AUC) of selected features, and new tools for identifying truly predictive features and for building sharper predictive models for phenotypes. He is particularly interested in uncovering the molecular mechanisms behind Alzheimer’s and Parkinson’s diseases. 


  
</ul>

<p>Many granting agencies including NSERC, CFI, CANSSI, CFREF, and MITACS have supported his research. To date, he has supervised the research of 3 postdoctoral fellows, 23 graduate students, and 14 undergraduate students.  His research findings have been published in prestigious journals such as the Journal of the American Statistical Association, Bayesian Analysis, Statistics in Medicine, Statistics and Computing, the American Statistician, the Journal of Applied Statistics, Scientific Reports, and BMC Bioinformatics. 

<p>His research activities are computationally intensive, aiming to develop new data analysis software and solve real problems in life and health sciences. He has published several R packages and functions publicly available on CRAN, GitHub, and his website.


<h2>Keys Words in His Research Interests </h2>

 statistical learning, cross-validation, hierarchical modelling, survival modelling, model checking, residual diagnostics, model comparison, zero-inflated models, high-throughput data, microbiome data



	

<h2 align="left">Externally Funded Research Projects</h2>

<ul>
<li>
	Statistical Methodologies and Computational Tools to Identify Microbial Correlates of Canadian Bee Gut Health,  <a href="https://canssi.ca/story/crt-29/">Collaborative Research Team Projects – Project 29</a>, Co-PI, 2025–2028.
</li>

<li>
	 Geospatial Artificial Intelligence Algorithms for Automating Manual Observation Associated with Wheat Production, MITACS Accelerate Grant, PI. 2022-2025.
</li>


<li> 
	Develop a web-based geospatial artificial intelligence framework to track, visualize, analyze, model, and predict infectious disease spread in real-time, MITACS Accelerate Grant, PI, 2020-2021. 
</li>

<li>
	<a href="https://cognit.ca/en/project/207670">Predictive Methods for Analyzing High-throughput and Spatial-temporal Data</a>, NSERC Individual Discovery Grant, 2019-2024, PI.
</li>

<li> 
	Genotype &amp; Environment to Phenotype, sub-project from Canada First Research Excellence Fund (CFREF) Project &quot;Designing Crops for Global Food Security&quot;, 2016-2019, Co-Investigator (PI: Prof. Kusalik).
</li>



<li> 
	Applications of Neural Network Curve Fitting Methods for Least-squares Monte Carlo Simulations in Financial Risk Management, MITACS Accelerate Internship Fund, 2016, PI. 
</li>


<li> 
	<a href="https://cognit.ca/en/project/13450">Bayesian Methods for High-dimensional and Correlated Data</a>, NSERC Individual Discovery Grant,  2014 - 2019, PI.

</li>	

<li> 
	<a href="https://www.nserc-crsng.gc.ca/ase-oro/Details-Detailles_eng.asp?id=527993">Efficient Bayesian Analysis for Complex Models</a>, NSERC Individual Discovery Grant,  2009 - 2014, PI.
</li>

<li> 
	A Computer Cluster for Research on Efficient Bayesian Statistical Methods, CFI Leaders Opportunity Fund,  2009, PI.
</li>

<li> 
	Clustering Analysis for Detecting the Types of Vehicles, MITACS Accelerate Grant, 2008, Co-PI with Prof. Laverty. 
</li>
<!--
<p>
It is very useful to build an automatic computer system to recognize the types of vehicles passing a checkpoint given some easy-to-get data about the vehicles, such as the distances between axles, the weights on each axle. Such a system has many applications, for example, in monitoring traffic volumes and identifies the type of vehicle, which will be helpful in budgeting road maintenance costs. The main goal of this project is to develop a better methodology for cluster analysis with application to the vehicle detection problem. The simplest clustering technique is the K-means clustering. However, K-means clustering requires that the users supply with a number of clusters. X-means clustering may be an alternative method since it can detect the number of clusters with some simple criteria. But X-means would introduce more severe local mode problem. We will investigate a new method to overcome the problem by merging similar clusters after running X-means clustering.
</p>
-->	
</ul>



</div>
</body>
</html>
